packages/vscode/src/plugins/prisma-language-server/index.ts

```diff
@@ -77,19 +77,23 @@ const plugin: PrismaVSCodePlugin = {
       watcher = chokidar.watch(
         path.join(rootPath, '**/node_modules/.prisma/client/index.d.ts'),
         {
-          usePolling: false,
+          // ignore dotfiles (except .prisma) adjusted from chokidar README example
+          ignored: /(^|[\/\\])\.(?!prisma)./,
```

Comment:
Why do we have to ignore all dot files if we are only watching a glob for `**/node_modules/.prisma/client/index.d.ts`?

Reply:
See main description for screenshot, it appears that without this ignore the watcher is actually watching everything in .bin folder. Didn't try but could be that it matches all dot folders in node_modules and given that some frameworks use these dot folders (with many files in there) it could be quite a black hole.

Reply:
Ok, that I understand. 

But couldn't that mean that the first param we are sending in there is broken in some way and could have other unexpected side effects?

Reply:
The glob looks fine though `**/node_modules/.prisma/client/index.d.ts` so maybe an edge case with dot folders, they tend to have weird things happening to them.

Reply:
The initial `**/` looks like something that might get expanded to something very large, maybe that's the problem?